kiwi/xml_parse.py

```diff
@@ -3459,10 +3458,8 @@ def factory(*args_, **kwargs_):
         else:
             return user(*args_, **kwargs_)
     factory = staticmethod(factory)
-    def get_group(self): return self.group
-    def set_group(self, group): self.group = group
-    def get_secgroups(self): return self.secgroups
-    def set_secgroups(self, secgroups): self.secgroups = secgroups
+    def get_groups(self): return self.groups
+    def set_groups(self, groups): self.groups = groups
```

Comment:
This is unacceptably non-pythonic and is just horrid. Please use [property decorator](http://www.programiz.com/python-programming/property), instead.